CONTRIBUTING.md

```diff
@@ -234,7 +234,7 @@ _Please note: Plugin API interface signatures and supported plugin API version(s
 
 1. Based on the unreleased content, determine the new version number and update
    the [VERSION](VERSION) file.
-1. Run `./bin/prefill_changelog` to populate the [changelog](CHANGELOG.md) with
+1. Run `./bin/prefill_changelog $(cat VERSION)` to populate the [changelog](CHANGELOG.md) with
```

Comment:
@izgeri there's a danglingly cat

Reply:
I'm not sure I follow - the command you run is `./bin/prefill_changelog [version]`, and `$(cat VERSION)` echos the `[version]` to that command

Reply:
so this change updates the contributing file to make it easier to run the changelog filler

Reply:
🤦‍♂️ I parsed this completely wrong